### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by ubcent** [qualiabyte/code-stats#7]
Hi! Thank you for the great package. Do you have any plans to support windows? On my windows 10 machine I've got the following error:
```

 Filetype | Line count
-----------------------
FIND: Parameter format not correct
(node:13968) UnhandledPromiseRejectionWarning: Error: Command failed: find * -type f
FIND: Parameter format not correct

    at checkExecSyncError (child_process.js:618:11)
    at execSync (child_process.js:655:13)
    at findSourceFiles (C:\Users\Dmitrii_Bondarchuk\AppData\Roaming\npm\node_modules\code-stats\code-stats.js:131:16)
    at printCounts (C:\Users\Dmitrii_Bondarchuk\AppData\Roaming\npm\node_modules\code-stats\code-stats.js:168:21)
    at printSorted (C:\Users\Dmitrii_Bondarchuk\AppData\Roaming\npm\node_modules\code-stats\code-stats.js:127:3)
    at printBody (C:\Users\Dmitrii_Bondarchuk\AppData\Roaming\npm\node_modules\code-stats\code-stats.js:123:3)
    at main (C:\Users\Dmitrii_Bondarchuk\AppData\Roaming\npm\node_modules\code-stats\code-stats.js:244:3)
    at Object.<anonymous> (C:\Users\Dmitrii_Bondarchuk\AppData\Roaming\npm\node_modules\code-stats\code-stats.js:248:1)
    at Module._compile (internal/modules/cjs/loader.js:707:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:718:10)
(node:13968) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing ins
ide of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejectio
n id: 1)
(node:13968) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejection
s that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by TheBird956** [comment]
I'm on Windows 10 too and I have the same error

**Comment by ubcent** [comment]
I'm just wondering whether this component is still maintained?

**Comment by qualiabyte** [comment]
@ubcent @TheBird956 Hi there, thanks for your interest in using this tool!  Adding Windows support was something I meant to do for a while, but it required finding a way to replace a call to the Unix find command.

Last week I finally spent some time figuring out the best way to do this, and ended up using the [`globby.sync()`](https://www.npmjs.com/package/globby) method which supports recursive directory search, glob pattern matching, and specifying multiple paths.

This is included in the latest release version `0.3.1`, which should work better now under Windows.  Additionally, file counts and percentage statistics are now reported by default thanks to changes by @aaronbeall.

**Comment by ubcent** [comment]
@qualiabyte Thanks for the update 👍
